lib/mongorito.js

```diff
@@ -32,14 +31,30 @@ const Query = require('./query');
 * Main class, manages mongodb connection and collections
 */
 
+let mongoritoInstance = Symbol();
+const MONGORITO_ENFORCER = Symbol();
 class Mongorito {
+  constructor(enforcer){
+    if(MONGORITO_ENFORCER !== enforcer) {
+      throw new EnforcerError('Cannot directly construct a Mongorito');
+    }
+    this._collections = {};
+  }
+
+  static get instance(){
```

Comment:
This and the instance Symbol enforce a single mongoritoInstance, the assumed this instance was failing all over the place for me in Node.js 4.1+ so I switched to a real Singleton pattern which is what this is enforcing.